.github/workflows/native_s3_dlr.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.operating-system }}
     strategy:
       matrix:
-        operating-system: [macos-latest]
+        operating-system: [ macos-10.15 ]
```

Comment:
is macos-latest causing issue?
I think is good idea to build native code on a fixed mac version, but run test on latest

Reply:
latest is the same as macos-10.15 now. When macos-11 is stable, action would point the latest tag to it